Core/Exercises/Services/TestRunners/HaskellService.cs

```diff
@@ -0,0 +1,31 @@
+using System.Net.Http.Json;
+using Core.Exercises.Models;
+using Microsoft.Extensions.Logging;
+
+namespace Core.Exercises.Services.TestRunners;
```

Comment:
Inconsistency in calling it SolutionRunner somewhere, and TestRunner elsewhere.

We should agree on what we want to kill the service.